Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.py

```diff
@@ -112,10 +125,53 @@ def get_static_report(client: Client, **args) -> CommandResults:
 
     r = client._http_request("GET", f"samples/{sample_id}/reports/static")
 
+    score = 0
+    if 'analysis' in r:
+        if 'score' in r['analysis']:
+            score = map_scores_to_dbot(r['analysis']['score'])
+
+    indicator: Any
+    if 'sample' in r:
+        target = r['sample']['target']
+        if r['sample']['kind'] == "file":
+            # Static can include data on multiple files e.g. in case of .zip upload.
+            # sample.target identifies the actual analysis subject so only get
+            # the results for that file
+            for file in r['files']:
+                if file['filename'] == target:
+                    dbot_score = Common.DBotScore(
+                        indicator=file['sha256'],
+                        indicator_type=DBotScoreType.FILE,
+                        integration_name="Hatching Triage",
+                        score=score
+                    )
+                    indicator = Common.File(
+                        name=r['sample']['target'],
+                        sha256=file['sha256'],
+                        md5=file['md5'],
+                        sha1=file['sha1'],
+                        dbot_score=dbot_score
+                    )
```

Comment:
Please add to the yml file
In commands you changed the new outputs
You can see how to do it [here](https://xsoar.pan.dev/docs/integrations/context-standards-mandatory)

Reply:
If you want I can help you with that but I need writing permissions.

Reply:
```
- contextPath: File.Name
  description: The full file name (including file extension).
  type: String
- contextPath: File.MD5
  description: The MD5 hash of the file.
  type: String
- contextPath: File.SHA1
  description: The SHA1 hash of the file.
  type: String
- contextPath: File.SHA256
  description: The SHA1 hash of the file.
  type: String
- contextPath: DBotScore.Indicator
  description: The indicator that was tested.
  type: String
- contextPath: DBotScore.Type
  description: The indicator type.
  type: String
- contextPath: DBotScore.Vendor
  description: The vendor used to calculate the score.
  type: String
- contextPath: DBotScore.Score
  description: The actual score.
  type: Number
- contextPath: URL.Data
  description: The URL
  type: String
```
Need to add it on the yml file in the outputs of the commands:
"**Triage-get-static-report**" in line 278
"**Triage-get-report-triage**" in line 299